addons/spreadsheet_dashboard_im_livechat/data/dashboards.xml

```diff
@@ -4,6 +4,8 @@
     <record id="spreadsheet_dashboard_livechat" model="spreadsheet.dashboard">
         <field name="name">Live chat</field>
         <field name="spreadsheet_binary_data" type="base64" file="spreadsheet_dashboard_im_livechat/data/files/livechat_dashboard.json"/>
+        <field name="main_data_model_ids" eval="[(4, ref('im_livechat.model_im_livechat_channel'))]"/>
```

Comment:
this isn't the correct model, it's not possible to not have any channels. could we use the live chat sessions ?

Reply:
I was able to delete all the channels though. What technical model exactly you are talking about ?

Reply:
the question isn't that you were able to delete all the channels, but what do you have if you install the apps without demo data. When I install all dashboards without demo data, I have 2 channels created (admins and general).
I don't know discuss very well. Not sure what model to choose

Reply:
In that case, we can keep the original model `im_livechat.report.channel`

![image](https://github.com/user-attachments/assets/85fef705-b0de-4484-8202-7dc0c6e721e0)

Reply:
if it works with a standart install by adding `im_livechat.report.channel` it's good for me :)